x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/test_pipeline/test_pipeline_tabs/tab_documents/add_documents_accordion/add_documents_accordion.tsx

```diff
@@ -23,13 +23,13 @@ const i18nTexts = {
   addDocumentsButton: i18n.translate(
     'xpack.ingestPipelines.pipelineEditor.addDocumentsAccordion.addDocumentsButtonLabel',
     {
-      defaultMessage: 'Add documents from index',
+      defaultMessage: 'Add a test document from an index',
     }
   ),
   addDocumentsDescription: i18n.translate(
     'xpack.ingestPipelines.pipelineEditor.addDocumentsAccordion.contentDescriptionText',
     {
-      defaultMessage: 'Provide the index name and document ID of the indexed document to test.',
+      defaultMessage: `Provide the document's index and document ID.`,
```

Comment:
It looks like this is now enclosed in backticks.

Reply:
This is intentional - it's using Javascript [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to handle the apostrophe.